market/auction.cpp

```diff
@@ -1598,8 +1598,8 @@ int auction::submit_nolock(char *from, double quantity, double real_price, KEY k
 				gl_name(OBJECTHDR(this),myname,sizeof(myname)),quantity<0?"ask":"offer",
 				from);
 		}
-		return 1;
```

Comment:
Defer to review from @dchassin on this one.  Old market bid will now pass a fail, which seems correct, but not sure if that will break anything.